src/summarizedexperiment/utils.py

```diff
@@ -0,0 +1,80 @@
+from typing import Optional, Sequence, MutableMapping
+from functools import reduce
+import numpy as np
+import pandas as pd
+import scipy.sparse as sp
+
+from .SummarizedExperiment import SummarizedExperiment
+
+
+def concat(
+    ses: Sequence[SummarizedExperiment],
+    how: str = "outer",
+    metadata: Optional[MutableMapping] = None,
+) -> SummarizedExperiment:
+    """Concatenate SummarizedExperiment objects.
+    Assume we are working with RNA-Seq experiments for now.
+
+    Parameters
+    ----------
+    ses: Sequence[SummarizedExperiment]
+        A sequence of SummarizedExperiment objects.
+    how: str
+        Method to join SummarizedExperiment objects.
+    metadata: str
+        Metadata of the new SummarizedExperiment.
+
+    Returns
+    -------
+    concat_se: SummarizedExperiment
+        The concatenated SummarizedExperiment.
+    """
+
+    # we assume that sces have the same assay structure (same number of rows and cols)
+    new_assays = {}
+    assay_names = ses[0].assays.keys()
```

Comment:
makes an assumption that the names of assays in the first experiment are going to be present in all other SE's as well, probably not true

Reply:
should be flexible with assays being different in different objects. 

- also should only merge by assay names across SE's

Reply:
right, was going to ask you about this but forgot. got my answer now :)